docs/README.md

```diff
@@ -0,0 +1,34 @@
+LEDET-BIMS documentation
+==============================
+
+## Technical documentation for LEDET-BIMS Project.
+
+## Firstly you need to be in the base project directory where the management
+ command is run from.
+
+```docs/docs/```
```

Comment:
command is run from django-bims folder, and do `make shell`

Reply:
Thanks @ann26 updated